### ISSUE TITLE
[ally-defect-4]: "if known" doesn't align with Design System standards

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#63707]
## Point of contact
Jamie Klenetsky Fay 

## Definition of done
1. Review and acknowledge feedback.
2. Fix and/or document decisions made.
3. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.
## Severity level
4

## Details
This is an issue with a field label on [Step 1 of 2: Applicant Information](https://staging.va.gov/careers-employment/education-and-career-counseling/apply-career-guidance-form-28-8832/claimant-information).

![Screenshot 2023-08-15 at 4 18 13 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/c259f8e0-a4ca-4186-af9e-40a62c3d4b75)

"Your VA file number (*If known)" doesn't follow [design system guidelines](https://design.va.gov/patterns/ask-users-for/social-security-number#usage). And it's confusing for users to see the "required" visual pattern - red text with an asterisk - next to a field that isn't required.

## Reproduction steps
1. Go to the [unauthenticated form](https://staging.va.gov/careers-employment/education-and-career-counseling/apply-career-guidance-form-28-8832/claimant-information)
2. Scroll down to "Your VA file number"

## Proposed solution or next steps
Follow the design system guidelines for [VA file number](https://design.va.gov/patterns/ask-users-for/social-security-number#usage).

- If it's required, use the standard (*Required) pattern.
- If it's not required, use a label like "VA file number (if known)," no red text. We also suggest adding helper text to explain why it's helpful for the Veteran to provide the VA file number, even though its optional.

## Type of issue
- [ ] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [x] Color
- [ ] Zoom
- [x] Semantics
- [ ] Axe-core
```[tasklist]
### Tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/69241
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/68001
```

**Comment by sara-amanda** [comment]
Connected to Epic verified. Removing `Sitewide CAIA` label.

**Comment by Midge-dev** [comment]
@coforma-jamie I took a look at the code for this as well as the physical form (`VBA-25-8832-ARE.pdf`) - the physical from _does_ ask for VA file number `if applicable` so the second option in the guidelines you listed would be optimal. We might need input somewhere on what to include in the suggested helper text, but for now we can update text or remove it altogether.

cc: @steele-lm @tblackwe

**Comment by steele-lm** [comment]
Confirmed solution with Julie P.
